### PR TITLE
Cleanup/manifest

### PR DESCRIPTION
**Pull request by jbout**

### CODE REVIEW

File: config/default/LockService.conf.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+/**
```
Comment (boajer) [inline config/default/LockService.conf.php:3]: should we have headers for the default configs stored separately?

File: config/default/LockService.conf.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ * Default config header created during install
+ */
+
+return new oat\oatbox\mutex\LockService(array(
```
Comment (boajer) [inline config/default/LockService.conf.php:7]: what about shor array notation?

File: config/default/LockService.conf.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    'persistence_class' => 'oat\\oatbox\\mutex\\NoLockStorage'
```
Comment (boajer) [inline config/default/LockService.conf.php:8]: Any reason of usage literals but not constants?

File: manifest.php [hunk 4]
```diff
@@ -23,6 +23,7 @@
  */
 
 use oat\generis\scripts\install\SetupDefaultKvPersistence;
+use oat\generis\scripts\install\TaskQueue;
```
Comment (boajer) [inline manifest.php:26]: Should we change copyright year at the very top of this file to 2020?

Comment (gabrielfs7) [reply manifest.php:26]: off: How you generally keep the copyright of the files updated? Only when there is a file modification? If we do not have an automatic way to do it, maybe it should be included in the Michael's checklist.

**Pull request merged**